### ISSUE TITLE
setup fails on TASK [api-dstu2-code : checkout repository hspc_repo_api_mysql_project] (branch 1.0)

### ISSUE THREAD

**Issue by gbtorrance** [smart-on-fhir/installer#14]
The Ansible playbook for branch 1.0 has begun failing on TASK [api-dstu2-code : checkout repository hspc_repo_api_mysql_project]. More specifically, it is prompting for login information for bitbucket. It seems that the repo https://bitbucket.org/hspconsortium/reference-api-mysql-deprecated is no longer publicly accessible. Is branch 1.0 supposed to still be usable? Thanks.

![20170614_smartsandboxansibleerror](https://user-images.githubusercontent.com/16735709/27135636-090166a0-50e7-11e7-905a-621a0cfc59ad.PNG)

**Comment by travistcummings** [comment]
Hi,

Can you please try out the 1.1.0 release?  It uses the HSPC Reference API release artifacts instead of the maven repository sourcecode.

Thanks!

**Comment by gbtorrance** [comment]
@travistcummings Unfortunately, it does not work for me. I checked out the 1.1.0 tagged code, copied the inventory_example.yml to inventory.yml, edited inventory.yml to set the following:

installer_user: "greg"
services_host: "xx.xxx.xxx.xxx"

... and then ran:

sudo ansible-playbook -c local -i 'inventory.yml,' -vvvv site.yml

I got the following error. I also asked someone else on the team to try, and he got the same error, too.

Thoughts? Should I create a new Issue for this?

Thanks,
Greg

![20170621_smartsandboxansibleerror](https://user-images.githubusercontent.com/16735709/27380483-4d45023e-564d-11e7-9012-41a407c54ba6.PNG)
